ProxyApp/Classes/CurrencyRate.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace ProxyApp
+{
+    class CurrencyRate
+    {
+        public int Cur_ID { get; set; }
+        public DateTime Date { get; set; }
+        public string Cur_Abbreviation { get; set; }
+        public int Cur_Scale { get; set; }
+        public string Cur_Name { get; set; }
+        public decimal Cur_OfficialRate { get; set; }
```

Comment:
1. The properties can be annotated with JsonPropertyNameTttribute which indicate the name of the json property name, so the class's properties can be named due to naming conventions.
2. Class members should be separated with empty line